ecs-cli/modules/cli/local/localproject/project.go

```diff
@@ -105,6 +106,7 @@ func (p *localProject) ReadTaskDefinition() error {
 
 	if remote != "" {
 		taskDefinition, err = p.readTaskDefinitionFromRemote(remote)
+		fmt.Printf("Reading task definition from %s:%v\n", aws.StringValue(taskDefinition.Family), aws.Int64Value(taskDefinition.Revision))
```

Comment:
Can you use `logrus.Info` instead of `fmt.Print` that's what we've been using in other commands

Reply:
done!